graphql-java-spring.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can you please remove this file?

Reply:
Yes, this was added by accident. It's removed now.